src/sentry/demo/demo_org_manager.py

```diff
@@ -112,9 +112,9 @@ def assign_demo_org() -> Tuple[Organization, User]:
         from .tasks import build_up_org_buffer
 
         demo_org = None
-        # option to skip the buffer when testing things out locally
-        if settings.DEMO_NO_ORG_BUFFER:
-            org = create_demo_org()
+        # option to skip the buffer when testing things out
+        if skip_buffer:
+            org = create_demo_org(quick=True)
```

Comment:
adding `quick=True` here since we need it to be fast (previously, I just changed the parameters for org generation so it's always really fast but that won't work outside my dev env).